packages/platform/atoms/event-types/hooks/usePlatformTabsNavigations.tsx

```diff
@@ -74,28 +74,29 @@ export const usePlatformTabsNavigations = ({ formMethods, eventType, team, tabs
       });
     }
 
-    navigation.splice(1, 0, {
-      name: "availability",
-      onClick: () => setCurrentTab("availability"),
-      isActive: currentTab === "availability",
-      href: `${url}?tabName=availability`,
-      icon: "calendar",
-      info:
-        isManagedEventType || isChildrenManagedEventType
-          ? formMethods.getValues("schedule") === null
-            ? "members_default_schedule"
-            : isChildrenManagedEventType
-            ? `${
-                formMethods.getValues("scheduleName")
-                  ? `${formMethods.getValues("scheduleName")} - ${t("managed")}`
-                  : `default_schedule_name`
-              }`
-            : formMethods.getValues("scheduleName") ?? `default_schedule_name`
-          : formMethods.getValues("scheduleName") ?? `default_schedule_name`,
-    });
+    tabs.includes("availability") &&
+      navigation.splice(1, 0, {
+        name: "availability",
+        onClick: () => setCurrentTab("availability"),
+        isActive: currentTab === "availability",
+        href: `${url}?tabName=availability`,
+        icon: "calendar",
+        info:
+          isManagedEventType || isChildrenManagedEventType
+            ? formMethods.getValues("schedule") === null
+              ? "members_default_schedule"
+              : isChildrenManagedEventType
+              ? `${
+                  formMethods.getValues("scheduleName")
+                    ? `${formMethods.getValues("scheduleName")} - ${t("managed")}`
+                    : `default_schedule_name`
+                }`
+              : formMethods.getValues("scheduleName") ?? `default_schedule_name`
+            : formMethods.getValues("scheduleName") ?? `default_schedule_name`,
+      });
 
     // If there is a team put this navigation item within the tabs
-    if (team) {
+    if (team && tabs.includes("assignment")) {
```

Comment:
`assignment` tab comes in as `?tabName=team`

![Screenshot 2024-11-05 at 2 22 12 PM](https://github.com/user-attachments/assets/a6f843b2-4f7f-4636-a5d7-24f3471c72ee)

Reply:
indeed, thank you 👍